index.html

```diff
@@ -4677,98 +4722,165 @@ <h2>Internationalization Considerations</h2>
 specification and significantly diminish its value as a standard.
     </p>
 
+    <p>
+Implementers are strongly advised to read the Requirements for
```

Comment:
Note that the current title of this document is "Strings on the Web: Language and Direction Metadata".

Reply:
Fixed.